src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/FindBadGenomicKmersSpark.java

```diff
@@ -32,6 +36,9 @@
         programGroup = StructuralVariationSparkProgramGroup.class)
 @BetaFeature
 public final class FindBadGenomicKmersSpark extends GATKSparkTool {
+    public static final int REF_RECORD_LEN = 10000;
```

Comment:
I agree, but since this is the code already in place in master, and after I moved back the intended-to-be-refactored out method back here, it is not used by any other code, I'll say let's keep it as is.